### DISCUSSION TITLE
How to stop sound when app is minimized ?

### DISCUSSION THREAD

**Discussion by lorydi3001** [bluefireteam/audioplayers#1391] [Q&A]
I added music to my game. When I minimize the app on the android device the sound keeps playing. Sometimes it crashes if left looping in background but I think this is a known bug. 

How can I make it stop when the app is not in foreground and resume when opened back? I thought that this was the default behavior. 

Thank you

**Comment by Gustl22** [comment]
Unfortunately I haven't tested or looked into this use case. Maybe you can check for the lifecycle via [WidgetsBindingObserver](https://api.flutter.dev/flutter/widgets/WidgetsBindingObserver-class.html) and stop it, when it goes to the background.
Let me know it this works for you.
